regression/tests/studio/test_course_outline.py

```diff
@@ -53,3 +55,28 @@ def test_edit_start_button(self):
         """
         self.studio_course_outline.click_edit_start_date_button()
         self.schedule_page.wait_for_page()
+
+    def test_content_visibility_help(self):
+        """
+        Scenario: Help link for
+        'Learn more about content visibility settings' is working
+        Given that I am on the 'Course Outline' page.
+        And I want help about the content visibility settings.
+        And I click 'Learn more about content visibility settings'
+        in the sidebar
+        Then Help link should open.
+        And help url should contain 'developing_course/
```

Comment:
nit: I think you can make this docstring more generic. (So that if we change the link, you only need to change the code, not the docstring.) You can say something like "And url should point to the content visibility help page"